docs/source/release_notes.rst

```diff
@@ -3,19 +3,27 @@
 Release Notes
 -------------
 
-.. Future Release
+  Future Release
   ==============
     * Enhancements
     * Fixes
     * Changes
     * Documentation Changes
+      * Update release notes and release format
```

Comment:
I guess we aren't checking for this yet in CI, but would be good to mention the PR number for all the release notes entries we have added, like we do in other libraries.